### ISSUE TITLE
More numpy-like operations for variables

### ISSUE THREAD

**Issue by nvaytet** [scipp/scipp#1094]
Here is a list of operations that could be nice to have in Scipp.
Some are pretty standard (Simon: now tracked in #1221), others are operations that are missing from numpy:

- [ ] `finmin` and `finmax`: "finite min/max" - a version of `nanmin/max` that ignores both NaNs and Inf (missing from `numpy`)
-  [ ] `minmax`: returns the lower and upper bounds of a Variable in one go, iterating through the values only once (missing from `numpy`). I am not clear if this can easily be implemented as the current version of transform does not allow for multiple return values. Maybe it's possible to return a `pair`?
- [x] `zeros_like`, `ones_like`

The names of the operations are of course up for discussion.

**Comment by SimonHeybrock** [comment]
👍, but removed from 0.4 milestone since the focus is now on consolidating what we have and adding urgent things.

**Comment by SimonHeybrock** [comment]
I added the operations that *are* in numpy to #1221, would hesitate to add any others for now, until we have concrete use cases and good motivation.

**Comment by SimonHeybrock** [comment]
Added to 0.5 since after #1277 this appears to be the dominant factor in the waiting time until a 2D plot shows up.

**Comment by SimonHeybrock** [comment]
Closing as part of issue cleanup. Open a new issue if a specific operation is required.
